code/modules/reagents/reagent_containers/bottle.dm

```diff
@@ -40,7 +43,7 @@
 				filling.icon_state = "[icon_state]100"
 
 		filling.icon += mix_color_from_reagents(reagents.reagent_list)
-		overlays += filling
+		underlays += filling
```

Comment:
Then we'd have the bottle 'label' pixels getting the same colour as the reagents, making them useless again. 
![bottlelabels](https://cloud.githubusercontent.com/assets/12377767/18193898/f455b066-70af-11e6-8b13-66216ff2b938.png)

The underlays elegantly accomplish the desired graphical effect with the simplest sprite, and it makes sense to have the chemical filler 'beneath' the bottle icon as the filler is supposed to represent the volume of liquid inside (behind the face of) the bottle.
I just feel like it's not much worth the effort to tint the sprites up when the underlays (and the one additional line of code that comes with it) do the job